addons/web/static/tests/widgets/data_export_tests.js

```diff
@@ -1,16 +1,32 @@
 odoo.define('web.data_export_tests', function (require) {
 "use strict";
 
+const AbstractStorageService = require('web.AbstractStorageService');
+const BusService = require('bus.BusService');
+const CrashManager = require('web.CrashManager').CrashManager;
 const data = require('web.data');
 const framework = require('web.framework');
 const ListView = require('web.ListView');
+const LocalStorageService = require('web.LocalStorageService');
+const RamStorage = require('web.RamStorage');
 const testUtils = require('web.test_utils');
+const {AsyncJobService} = require('bus.AsyncJobService');
 
 const cpHelpers = testUtils.controlPanel;
 const createView = testUtils.createView;
 
+const TestBusService = BusService.extend({
+    TAB_HEARTBEAT_PERIOD: 10,
+    MASTER_TAB_HEARTBEAT_PERIOD: 1,
+});
+
+let LocalStorageServiceMock;
+
 QUnit.module('widgets', {
     beforeEach: function () {
+        LocalStorageServiceMock = AbstractStorageService.extend({storage: new RamStorage()});
+        var self = this;
```

Comment:
unused

Reply:
LocalStorageServiceMock is required but I moved it outside of beforeEach

Reply:
Comment is about `var self = this`. `LocalStorageServiceMock` looks used so it's good.